addons/autocomplete_googleplaces/models/website.py

```diff
@@ -0,0 +1,18 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    google_places_api_key = fields.Char('Google Places API Key', compute='_compute_places_api_key', inverse='_inverse_places_api_key')
```

Comment:
Once you have google places API enabled, you may create as many keys as you wish so that you can track usage of each keys. My train of thoughts here was that you could keep track of each website's request individually like this.
What do you think ?

Reply:
Ok makes sense. 👍